src/components/FooterContainer/index.tsx

```diff
@@ -1,61 +1,14 @@
 // SPDX-License-Identifier: MPL-2.0
 // Copyright (C) 2019 - 2021 Gemeente Amsterdam
-import {
-  Row,
-  Heading,
-  Link,
-  themeColor,
-  themeSpacing,
-  Footer,
-  FooterBottom,
-} from '@amsterdam/asc-ui'
+import { Link, Footer, FooterTop, FooterBottom } from '@amsterdam/asc-ui'
 import styled from 'styled-components'
 
 import configuration from 'shared/services/configuration/configuration'
-import stringFormatter from 'shared/services/stringFormatter'
```

Comment:
if the `stringFormatter` is no longer required then you can also remove the files

Reply:
✔️